tests/contract_testcases/substrate/account_info.sol

```diff
@@ -1,4 +1,4 @@
-contract c {
+abstract contract c {
```

Comment:
Why does this contract need to be abstract? I has public functions.

Reply:
good catch!